security.md

```diff
@@ -134,7 +134,11 @@ Using git would allow you to add outsource/freelance developers for a limited ti
 
 * There are extra protection products on top of an antivirus called EDR (Cyberreason, BlackCobalt) but these are usually costly.  
 
-### Physical security
+### Physical Security
+
+* Lock every workstation when you walk away from it.

```

Comment:
Do you mean moving the mouse to the top right corner or similar? 
Most employees are not that conscious about that, have you seen a company where all employees do that?
In most cases there is a policy which after X minutes shows the login screen.